bin/postman-test/collections/core-data.postman_collection.json

```diff
@@ -1270,6 +1270,56 @@
 						"description": "Remove all old events (and associated readings) based on delimiting age.  ServiceException (HTTP 503) for unknown or unanticipated issues.  Should only be used by the scrubber micro service"
 					},
 					"response": []
+				},
+				{
+					"name": "http://localhost:48080/api/v1/event event can be added with binary value",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "cb08cb30-ce8a-4b39-a04b-ac666350fb09",
+								"exec": [
+									"/*",
+									" * Test Case:  /api/v1/event - POST",
+									" *",
+									" */",
+									"tests[\"Status code is 200\"] = responseCode.code === 200;",
+									"tests[\"Response time is less than \"+data.responseTime+\"ms\"] = responseTime < data.responseTime;",
+									"var contentTypeHeaderExists = responseHeaders.hasOwnProperty(\"Content-Type\");",
+									"tests[\"Has Content-Type\"] = contentTypeHeaderExists;",
+									"if (contentTypeHeaderExists) {",
+									"    tests[\"Content-Type is text/plain\"] = responseHeaders[\"Content-Type\"].has(\"text/plain\");",
+									"}",
+									"if(null !== responseBody){",
+									"    tests[\"Response Object id\"] = responseBody.length === 36;",
+									"}"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"device\": \"irCamera\",\n    \"readings\": [\n        {\n           \"device\": \"irCamera\",\n           \"name\": \"image\",\n           \"binaryValue\": \"83010203\",\n           \"mediaType\": \"image\"\n        }\n    ]\n}"
+						},
+						"url": {
+							"raw": "http://localhost:48080/api/v1/event",
+							"protocol": "http",
+							"host": [
+								"localhost"
+							],
+							"port": "48080",
+							"path": [
+								"api",
+								"v1",
+								"event"
+							]
+						}
```

Comment:
Should use `{{baseUrl}}` for different environment
```
"url": {
        "raw": "{{baseUrl}}/api/v1/event",
        "host": [
                     {{baseUrl}}
        ],
        "path": [
                     "api",
                     "v1",
                     "event"
        ]
}
```

Reply:
Whoops, thank you.  Will fix.